detekt.yml

```diff
@@ -2,24 +2,35 @@ config:
   warningsAsErrors: true
 
 complexity:
+  ComplexMethod:
```

Comment:
I would prefer that we don't change these values just so our existing code passes. We should leave the thresholds as they were to encourage us to not create large, complicated and potentially unreadable code.